### PR TITLE
Add `2201.6.1` release note

### PR DESCRIPTION
**Pull request by chiranSachintha**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

**Review by NipunaRanasinghe — APPROVED** [review]
Tooling release notes LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Improvements
+
+#### OpenAPI Tool
+- Added support for object-typed query parameters in the OpenAPI to Ballerina service generation.
+- Added support to generate low-level service skeletons without any data-binding logic. This mode can be enabled by adding the `--without-data-binding` flag to the OpenAPI to Ballerina service generation command.
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed+label%3AType%2FBug).
```
Comment (lnash94) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:50]:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed+label%3AType%2FBug).
- [OpenAPI Tool](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed)
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.6.1+label%3AType%2FBug+is%3Aclosed).
+
+## Standard library updates
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.6.1+label%3AType%2FBug+is%3Aclosed+).
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:38]:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+milestone%3A2201.6.1+label%3AType%2FBug+is%3Aclosed+).
```
This is empty. Need to tag the issues to the milestone.

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Developer tools updates
+
+### Improvements
+
+#### OpenAPI Tool
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:44]:
```suggestion
#### OpenAPI Tool
```
Keep a new line between the title and the bullet points.

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md (hunk 1 above)
Comment (lnash94) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:50]:
```suggestion
To view bug fixes, see the [GitHub milestone for 2201.6.1 (Swan Lake)](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed+label%3AType%2FBug).
- [OpenAPI Tool](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed)
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md (hunk 2 above)
Comment (chiranSachintha) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:38]: Updated

**Review by lnash94 — APPROVED** [review]

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-6-1
+    - /downloads/swan-lake-release-notes/2201.6.1/
+    - /downloads/swan-lake-release-notes/2201-6-1-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-6-1-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.6.1
+
+<em>Swan Lake 2201.6.1 is the first patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language, tooling, and standard library.</em>
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:15]:
```suggestion
<em>Swan Lake 2201.6.1 is the first patch release of Ballerina 2201.6.0 (Swan Lake Update 6) and it includes a new set of bug fixes to the language,standard library, and  tooling.</em>
```

File: downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+#### OpenAPI Tool
+
+- Added support for object-typed query parameters in the OpenAPI to Ballerina service generation.
+- Added support to generate low-level service skeletons without any data-binding logic. This mode can be enabled by adding the `--without-data-binding` flag to the OpenAPI to Ballerina service generation command.
+
+### Bug fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.6.1 of the repository below.
+- [OpenAPI Tool](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+is%3Aclosed)
```
Comment (praneesha) [inline downloads/swan-lake-release-notes/swan-lake-2201.6.1/RELEASE_NOTE.md:52]:
```suggestion
- [OpenAPI Tool](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.6.1%22+label%3AType%2FBug+is%3Aclosed+)
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**